webapp/src/app/ui/pull-request-widget/pull-request-widget.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="border border-gray-300 rounded-lg p-4 w-72">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold">{{ pullRequest().repository.name }} #{{ pullRequest().number }} on {{ pullRequest().createdAt }}</span>
+    <span class="flex items-center space-x-2">
+      <span class="text-green-600 font-bold">+{{ pullRequest().additions }}</span>
+      <span class="text-red-600 font-bold">-{{ pullRequest().deletions }}</span>
+    </span>
+  </div>
+
+  <div class="flex justify-between font-bold text-sm mb-3">
+    {{ pullRequest().title }}
+    @if (pullRequest().reviews) {
+      <span class="text-xs text-gray-500">({{ pullRequest().reviews.size }} reviews)</span>
```

Comment:
Here you can just show number of approvals with an check icon, change requests, and comments.